lib/mini_portile2/mini_portile_cmake.rb

```diff
@@ -94,6 +94,9 @@ def find_c_and_cxx_compilers(host)
     if MiniPortile.darwin?
       c_compiler ||= 'clang'
       cxx_compiler ||='clang++'
+    elsif MiniPortile.freebsd?
+      c_compiler ||= 'cc'
+      cxx_compiler ||= 'c++'
```

Comment:
We could potentially add this to the branch above as `clang` and `clang++` are on the `PATH` too but I went with `cc` and `c++` as that's what [the official documentation uses](https://docs.freebsd.org/en/books/developers-handbook/tools/#tools-compiling).